server/server.js

```diff
@@ -125,6 +125,18 @@ module.exports = function(__dirname) {
 				value: body.content
 			}
 		}
+		if (!(body.tags.length > 1 || body.tags[0] || body.title || body.content) && modify) {
+			return new Promise(function(resolve, reject) {
```

Comment:
If we're not using a mongo client that supports promises, we should switch to one that does.